### PR TITLE
Handle delete

### PR DESCRIPTION
**Pull request by jacksonyuanjx**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -153,7 +153,10 @@ router.post("/post/ad", async (req, res) => {
 
 router.delete("/post/:id", async (req, res) => {
     try {
-        await queries.deletePost(req.params.id);
+        const result = await queries.deletePost(req.params.id);
+        if (result.rowCount === 0) {
+            return res.status(400).send(`Reply with postId = ${req.params.id} not found`);
```
Comment (jacksonyuanjx) [inline src/index.js:158]: @Nefertari-Qin  Could you change this message to reflect that it's a "post with postId..."?

File: src/index.js [hunk 2]
```diff
@@ -192,7 +195,10 @@ router.put("/reply", async (req, res) => {
 
 router.delete("/reply/:id", async (req, res) => {
     try {
-        await queries.deleteReply(req.params.id);
+        const result = await queries.deleteReply(req.params.id);
+        if (result.rowCount === 0) {
+            return res.status(400).send(`Reply with postId = ${req.params.id} not found`);
```
Comment (jacksonyuanjx) [inline src/index.js:200]: Similarly here, I think it should say "reply with replyId..."

File: src/index.js (hunk 1 above)
Comment (Nefertari-Qin) [reply src/index.js]: fixed, thx

File: src/index.js (hunk 2 above)
Comment (Nefertari-Qin) [reply src/index.js]: right, fixed, thank you again.

**Pull request merged**